src/features/categories/category-selection/category-badge/CategoryBadge.module.scss

```diff
@@ -26,3 +34,22 @@
     padding-left: var(--spacing-xs);
   }
 }
+
+.deleteIconContainer {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-left: var(--spacing-xs); /* Adjust margin to move closer to the badge */
+}
+
+.deleteIcon {
+  cursor: pointer;
+  transform: translateX(0); /* Remove the left shift */
+  color: var(--color-neutral-3); /* Match color to PlusCircle if needed */
+  transition: transform 0.2s ease-in-out;
+
+  &:hover {
```

Comment:
The app is designed for touch, there won't be hover effects. I appreciate the extra but this code can be removed.